### ISSUE TITLE
[BREAKING] Change repository name?

### ISSUE THREAD

**Issue by n1ngu** [ndeloof/install-compose-action#2]
Since the repository name will be what appears in the `use` block in any workflows using it, it might be interesting to:
- adopt the setup-tool naming convention https://docs.github.com/en/actions/creating-actions/developing-a-third-party-cli-action#introduction
- introduce the "2" keyword in the name to differentiate the action from former "docker compose setup" actions that can only install compose v1 series via pip and whatnot. I had a hard time triaging https://github.com/marketplace?type=actions&query=compose until I found this repo!

E.g: ndeloof/setup-docker-compose-2 ?

I think the earlier this is addressed the better for spreading the adoption of this action!

**Comment by ndeloof** [comment]
good points
To be considered if we want to make this an official action, aside https://github.com/docker/setup-buildx-action

**Comment by felipecrs** [comment]
I don't know if this is indeed a breaking change. I mean, GitHub setups a redirect for the old repo name anyway. Unless GH Actions does not follow the redirect.

**Comment by n1ngu** [comment]
> I don't know if this is indeed a breaking change. I mean, GitHub setups a redirect for the old repo name anyway. Unless GH Actions does not follow the redirect.

Yeah, I meant it as "breaking if actions wouldn't follow redirects" but hopefully non-breaking.

**Comment by asset-web** [comment]
If the README page could be updated to use:

`ndeloof/install-compose-action`

not 

`ndeloof/install-docker-compose`

as this does not work, that would be helpful.
